Docs/Configuration.tex

```diff
@@ -265,55 +265,64 @@ \subsection{Configuration Terms}\label{configuration-terms}
   \texttt{invalid\ value} if missing.
 \item
   \texttt{fatal\ behaviour} --- behaviour leading to boot termination.
-  Implementation must stop the boot process from going any further until
-  next host system boot. It is allowed but not required to perform cold
-  reboot or show any warning message.
+  Implementations shall prevent the boot process from continuing until
+  the next host system boot. It is permitted, but not required, to
+  execute cold reboots or to show warning messages in such cases.
 \item
   \texttt{undefined\ behaviour} --- behaviour not prescribed by this
-  document. Implementation is allowed to take any measures including but
-  not limited to \texttt{fatal\ behaviour}, assuming any states or
-  values, or ignoring, unless these measures negatively affect system
-  security in general.
+  document. Implementations may take any measures including, but not
+  limited to, measures associated with \texttt{fatal\ behaviour}, assumptions
+  of any state or value, or disregarding any associated states or values. This is
+  however subject to such measures not negatively impacting upon system integrity.
 \end{itemize}
 
 \subsection{Configuration Processing}\label{configuration-processing}
 
-\texttt{OC\ config} is guaranteed to be processed at least once if it
-was found. Depending on OpenCore bootstrapping mechanism multiple
-\texttt{OC\ config} files may lead to reading any of them. No
-\texttt{OC\ Config} may be present on disk, in which case all the values
-read follow the rules of \texttt{invalid\ value} and
-\texttt{optional\ value}.
+The \texttt{OC\ config} file is guaranteed to be processed at least once if found.
+Depending on OpenCore bootstrapping mechanism, the presence of multiple
+\texttt{OC\ config} files may lead to the reading of any of them. It is permissible
+for no \texttt{OC\ Config} file to be present on disk. In such cases, all values
```

Comment:
`In such cases, if the implementation does not abort the boot process, all values…` This is how it actually happens now, and I believe we need to give room.